NitroxClient/GameLogic/LiveMixinManager.cs

```diff
@@ -8,7 +7,7 @@ public class LiveMixinManager
 {
     private readonly SimulationOwnership simulationOwnership;
 
-    private bool processingRemoteHealthChange = false;
+    public bool ProcessingRemoteHealthChange { get; private set; } = false;
```

Comment:
Default for booleans is `false`, no need to init with `false`.
Recommend renaming to something more to-the-point:
```suggestion
    public bool IsRemoteHealthChanging { get; private set; }
```